csharp/src/SeedLang/Visualization/Value.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using SeedLang.Runtime;
+
+namespace SeedLang.Visualization {
+  // A wrapper class of the underlying VMValue. It's used to represent runtime values in the
```

Comment:
I like this design to separate the internal representation (VMValue) and the external representation (Value).